platforms/Windows/python/asserts/python.wxs

```diff
@@ -0,0 +1,14 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <?define VariantName = asserts ?>
```

Comment:
Do we need the `VariantName` now?

Reply:
Removed it, it was only used for the license.